qa/automation/CareerPortal.Tests.Scenarios/Sataeva_cv form_feature/CV_form.feature

```diff
@@ -0,0 +1,181 @@
+Feature: Send CV form
```

Comment:
About the very beginning of this file: you need to specify the file number. For example, 1.2.1 Send CV. feature.

Reply:
I copy-pasted somehow the previous version without numeration.
Will add now.